index.js

```diff
@@ -245,6 +250,17 @@ module.exports = class Hyperdrive extends ReadyResource {
     return await this.entry(name) !== null
   }
 
+  watch (folder) {
+    folder = normalizePath(folder || '/')
+
+    if (folder.endsWith('/')) folder = folder.slice(0, -1)
+
+    const range = this.sub.range({ gt: folder + '/', lt: folder + '0' })
```

Comment:
use latest major of subencder (this api is gone)

Reply:
also just make the subEncoder here instaed of storing it. when @HDegroote's tweaks land we can just use his encoder